soh/include/z64save.h

```diff
@@ -184,7 +184,6 @@ typedef struct {
     char ganonText[250];
     u8 seedIcons[5];
     u16 randomizerInf[2];
```

Comment:
this is no longer big enough

Reply:
Great catch, I made a mental note to do this and forgot :<